shared/data/utils/challenges.helper.ts

```diff
@@ -11,3 +11,9 @@ export function getSortedChallengesByDifficulty(challenges: Map<string, IChallen
   });
   return sortedChallengesByDifficulty;
 }
+
+//TODO: make filter function that takes key to search in & value to search for, and boolean for case-insensitive
+export function getSortedChallengesByTitle(challenges: Map<string, IChallenge>, value: string) {
```

Comment:
name is confusing.
Function does filtering but name says sorting

Reply:
https://github.com/sadanandpai/frontend-mini-challenges/pull/396/commits/e4d42435534cbd14bc9fd3c20a83d9641677f091